### ISSUE TITLE
Custom validation fails even if properties meet the requirements

### ISSUE THREAD

**Issue by DimitrisRK** [tjinauyeung/svelte-forms-lib#125]
### Summary

I use `validateFunction` without the use of yum validator and validation seems to fail if properties are inside nested objects.
Example:
```
{
    name: "test", // This validates
    location: { // This does not work, as 'isEmpty' check considers it as an input value and not as a subset of properties
       lat: 0,
       long: 0
    }
}
```

### Steps to reproduce

Create a validateFunction that returns an object schema with nested properties.

### What is the current *bug* behavior?

Validation fails on a complicated schema.
It seems that function `isEmpty` considers properties that have an object as a value as invalid which is wrong.

### What is the expected *correct* behavior?

Validator should skip validation of properties that have an object as a value.

### Possible fixes

Replace function `util.isEmpty` check with `util.getValues().length`.

**Comment by larrybotha** [comment]
@DimitrisRK thanks for the PR! Your change looks good, but the tests don't cover a regression for this issue. Please add a test for this, something along the lines of (around line 490 would make sense):

```javascript
    it('validateFn handles nested values', done => {
      const value = 'bar';
      const event = {
        target: {
          name: 'nested.foo',
          value,
        },
      };
      const instance = createForm({
        initialValues: {
          nested: {
            foo: '',
          },
        },
        validate: values => values,
        onSubmit: values => console.log(values),
      });

      instance
        .handleChange(event)
        .then(errors => expect(errors.nested.foo).toBe(''))
        .then(done);
    });

```

Please check that the test fails when your fix isn't applied. Let me know if you have any questions!

**Comment by DimitrisRK** [comment]
@larrybotha I added a test for this case. Thanks for pointing out.

**Comment by larrybotha** [comment]
@DimitrisRK thanks!

Issue closed with #126
